client/a8c-for-agencies/style.scss

```diff
@@ -414,3 +414,18 @@ html {
 		}
 	}
 }
+
+.gray-tooltip {
+	.popover__arrow {
+		&::before {
+			border-bottom-color: var(--studio-gray-60) !important;
```

Comment:
If possible, can we avoid using `!important` here?

Reply:
Sorry, I copied this code from another place, so I missed this.